.github/workflows/main.yml

```diff
@@ -32,14 +30,16 @@ jobs:
         uses: actions/checkout@v2.3.1
 
       - name: "Install JDK"
-        uses: actions/setup-java@v1.3.0
+        uses: actions/setup-java@v1.4.3
         with:
           java-version: "11.0.5"
 
       - name: "Download Bazelisk"
         run: |
-          curl --location --fail "https://github.com/bazelbuild/bazelisk/releases/download/v${VERSION_BAZELISK}/bazelisk-linux-amd64" --output /tmp/bazel
-          chmod +x /tmp/bazel && echo "::add-path::/tmp/"
+          curl -LO "https://github.com/bazelbuild/bazelisk/releases/download/v1.5.0/bazelisk-linux-amd64"
+          mkdir -p "${GITHUB_WORKSPACE}/bin/"
+          mv bazelisk-linux-amd64 "${GITHUB_WORKSPACE}/bin/bazel"
+          chmod +x "${GITHUB_WORKSPACE}/bin/bazel"
```

Comment:
Can I ask you to attempt a different approach? Instead of all changes above (including the `env` removal):

```diff
- chmod +x /tmp/bazel && echo "::add-path::/tmp/"
+ chmod +x /tmp/bazel && echo "/tmp/" >> $GITHUB_PATH
```
This follows [GitHub recommendations](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#adding-a-system-path) and allows to use binaries without a risk of forgetting to use a correct path. For example, `bazel` invocations below were not changed, resulting in using a system-installed Bazel, not Bazelisk.

Reply:
Updated everything

Reply:
Awesome, thanks!